.gitlab-ci.yml

```diff
@@ -47,8 +47,11 @@ stages:
     - sysctl -w net.ipv4.tcp_keepalive_time=60
     - sysctl -w net.ipv4.tcp_keepalive_intvl=60
     - sysctl -w net.ipv4.tcp_keepalive_probes=10
+
   script:
-    # upload artifacts
+    # Upload flink shaded jackson artifacts
+    - su -c "cd flink-shaded-jackson-parent && $MVN $MAVEN_CLI_COMMON_OPTS -Pdocs-and-source -DaltDeploymentRepository=releases.repo.splunk.com::default::https://repo.splunk.com/artifactory/maven-splunk-local $MAVEN_CLI_DEPLOY_OPTS $CUSTOM_JOB_OPTS deploy"
```

Comment:
We also need to install this artifact before we build. Deploy always runs through a build.

Reply:
Thanks! I added back the install script

Reply:
After checking. It is not necessary because install is run as part of deploy

Reply:
In any case, we must make sure that we run install before the main Flink build. The jackson project and the Flink project separate Maven projects, so we must make sure to run the Jackson project first.